packages/ai-jsx/src/experimental/next/index.ts

```diff
@@ -6,10 +6,15 @@ import { asJsxBoundary } from '../../react/jsx-boundary.js';
 import { AIJSXError, ErrorCode } from '../../core/errors.js';
 import { toSerializedStreamResponse } from '../../stream/index.js';
 import { ComponentMap } from '../../react/map.js';
+import { Image } from '../../core/image-gen.js';
 export * from '../../react/core.js';
 
 function unwrapReact(partiallyRendered: AI.PartiallyRendered): ReactModule.ReactNode {
   if (AI.isElement(partiallyRendered)) {
+    if (partiallyRendered.tag === Image) {
+      return ReactModule.createElement('img', { src: partiallyRendered.props.url });
```

Comment:
should we be using https://nextjs.org/docs/pages/api-reference/components/image here?

Reply:
Here, perhaps. But I don't think for the others, which are either hydrated or running on the client. (But I also don't totally know all that the NextJS Image does.)

Reply:
can dalle just return the data as bytes? we can then use a data URI.

Reply:
Tried fixing the issue with the help of Nick, but we still couldn't figure it out. We decide to use the `<img />` tag for now and figure out `Image` later if need be.